locustfile.py

```diff
@@ -0,0 +1,205 @@
+            headers={'Authorization': 'Bearer ' + self.access_token},
+            json={
+                'old_list_name': 'load_test_list',
+                'new_list_name': 'load_test_list2'
+            }
+        )
+
+        self.client.delete(
+            '/api/v3/user/lists/load_test_list2',
+            headers={'Authorization': 'Bearer ' + self.access_token}
+        )
+
+        # Recipies
+        self.client.get(
+            '/api/v3/user/recipe',
+            headers={'Authorization': 'Bearer ' + self.access_token}
+        )
+
+        self.client.post(
+            '/api/v3/user/recipe/load_test_recipe',
+            headers={'Authorization': 'Bearer ' + self.access_token}
+        )
+
+        self.client.get(
+            '/api/v3/user/recipe/load_test_recipe',
+            headers={'Authorization': 'Bearer ' + self.access_token}
+        )
+
+        self.client.post(
+            '/api/v3/user/recipe/load_test_recipe/ingredient',
+            headers={'Authorization': 'Bearer ' + self.access_token},
+            json={
+                'list_name': 'load_test_list',
+                'ingredient': 'Chewing gum',
+                'amount': 500,
+                'unit': 'g',
+                'is_custom_ingredient': False
+            }
+        )
+
+        url_string = '/api/v3/user/recipe/load_test_recipe/ingredient'
+        url_string += '?ingredient=Chewing gum'
+        url_string += '&unit=g&is_custom_ingredient=false'
+        self.client.delete(
+            url_string,
+            headers={'Authorization': 'Bearer ' + self.access_token}
+        )
+
+        self.client.post(
+            '/api/v3/user/recipe/load_test_recipe/step',
+            headers={'Authorization': 'Bearer ' + self.access_token},
+            json={'step': 'Load test step.'}
+        )
+
+        self.client.patch(
+            '/api/v3/user/recipe/load_test_recipe/step',
+            headers={'Authorization': 'Bearer ' + self.access_token},
+            json={'step': 'Load test step 2.', 'step_number': 1}
+        )
+
+        self.client.delete(
+            '/api/v3/user/recipe/load_test_recipe/step?step_number=1',
+            headers={'Authorization': 'Bearer ' + self.access_token}
+        )
+
+        self.client.delete(
+            '/api/v3/user/recipe/load_test_recipe',
+            headers={'Authorization': 'Bearer ' + self.access_token}
+        )
+
+    """
+    on_start
+
+    Preforms initial login when runner is created
+    """
+    def on_start(self):
+        global user_count
+        user_count = user_count + 1
+        self.user_id = user_count
+        self.access_token = ''
+
+        if (self.access_token == ''):
+            load_env_variables()  # Adds the .env values to environment to allow use
+
+            # Log in user
+            print_str = 'Logging User In as user: '
+            print_str += str(os.environ['LOAD_TEST_USERNAME'])
+            print_str += '_'+str(self.user_id)+'@gmail.com'
```

Comment:
add a space between the operators here like we do in other places. Specifically, like `+ str(self.user_id)`